src/less/dialog/ds6/dialog-base.less

```diff
@@ -40,8 +40,10 @@
             -webkit-overflow-scrolling: touch;
 
             .dialog__close {
+                /* csslint ignore:start */
                 position: -webkit-sticky;
```

Comment:
Thinking about it, shouldn't autoprefixer add the -webkit-sticky in the compiled CSS? Then the lesshint warning goes away.